src/js/nav/buildNav.js

```diff
@@ -0,0 +1,74 @@
+const yaml = require('js-yaml');
+const axios = require('axios');
+const _ = require('lodash');
```

Comment:
No need to import entire lodash, just those things that you are using
```
const { has } = require('lodash');
```

Given that only `has` is used no need to import lodash at all.